### ISSUE TITLE
Error on https://ruffle.rs/demo/

### ISSUE THREAD

**Issue by Karichain** [ruffle-rs/ruffle#8383]
# Error Info
Error name: Error
Error message: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:573:14
Error stack:
```
Error: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:573:14
    at n.wbg.__wbg_new_f9bb493369a83c12 (https://ruffle.rs/demo/966.index.js:1:6474)
    at std::sync::once::Once::call_once::{{closure}}::h892972e0a1d8fcda (wasm://wasm/018de1f6:wasm-function[1295]:0x2d3056)
    at std::sync::once::Once::call_inner::hb30cfbde5209eed0 (wasm://wasm/018de1f6:wasm-function[1581]:0x3242fd)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hc905d90a5e7f3ca1 (wasm://wasm/018de1f6:wasm-function[5124]:0x4a6fdc)
    at std::panicking::rust_panic_with_hook::hb09154fa23e06c37 (wasm://wasm/018de1f6:wasm-function[3756]:0x46143f)
    at core::panicking::panic_fmt::h9e229748e3ae9f9d (wasm://wasm/018de1f6:wasm-function[5358]:0x4b0918)
    at core::option::expect_failed::ha758e0d1c5165b94 (wasm://wasm/018de1f6:wasm-function[6088]:0x4c3de3)
    at ruffle_core::display_object::stage::Stage::root_clip::h92c3c9c1f81b292c (wasm://wasm/018de1f6:wasm-function[3729]:0x45ed5e)
    at ruffle_web::Ruffle::tick::h50545bb64813cc21 (wasm://wasm/018de1f6:wasm-function[716]:0x1f01d9)
    at ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::hf39a69090e41d86f (wasm://wasm/018de1f6:wasm-function[6319]:0x4cb417)
```

# Player Info
Allows script access: false
Renderer: WebGL

# Page Info
Page URL: https://ruffle.rs/demo/

# Browser Info
User Agent: Mozilla/5.0 (X11; CrOS x86_64 14695.85.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36
Platform: Linux x86_64
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-10-25
Channel: nightly
Built: 2022-10-25T00:41:37.332Z
Commit: 014982d18e205d86d84eb9cf484ec6bb9a643224
Is extension: false

# Metadata
width: 800
height: 681
frameRate: 30
numFrames: 5
swfVersion: 19
backgroundColor: undefined
isActionScript3: true
uncompressedLength: 68137

**Comment by Karichain** [comment]
sfh2

**Comment by n0samu** [comment]
I assume you mean Strike Force Heroes 2?

**Comment by Karichain** [comment]
yes

**Comment by Karichain** [comment]
hello

**Comment by n0samu** [comment]
I just wanted to verify what game you were having the issue with. This is a common issue with many ActionScript 3 games and I don't think it will be solved soon.

**Comment by Karichain** [comment]
can u make it better

**Comment by Karichain** [comment]
gg

**Comment by Karichain** [comment]
get good

**Comment by Dinnerbone** [comment]
Crash was fixed in #10026.
